lib/twilio-ruby/jwt/access_token.rb

```diff
@@ -69,6 +69,11 @@ def payload
           payload
         end
 
+        class << self
```

Comment:
Isnt the ConversationsGrant different from the VideoGrant? Not sure we want to be modifying it.